lib/deprecation-cop-status-bar-view.coffee

```diff
@@ -22,6 +22,13 @@ class DeprecationCopStatusBarView extends View
     @subscriptions.add atom.packages.onDidUnloadPackage @updateDeprecatedSelectorCount
     @subscriptions.add atom.packages.onDidActivatePackage @updateDeprecatedSelectorCount
 
+    @subscriptions.add atom.keymaps.onDidReloadKeymap (event) =>
+      @updateDeprecatedSelectorCount() if event.path is atom.keymaps.getUserKeymapPath()
+
+    try
+      userStylesheetFile = new File(atom.styles.getUserStyleSheetPath())
```

Comment:
Just want to throw this out there, instead of using `File` here you might be able to do this via the `atom.styles` API.

Something like:

```coffee
userStylesheetPath = atom.styles.getUserStyleSheetPath()
stylesChanged = (element) =>
  @updateDeprecatedSelectorCount() if element.getAttribute('source-path') is userStylesheetPath
atom.styles.onDidUpdateStyleElement(stylesChanged)
atom.styles.onDidAddStyleElement(stylesChanged)
```